dln.c

```diff
@@ -298,12 +302,12 @@ COMPILER_WARNING_POP
 /* assume others than old Mac OS X have no problem */
 # define dln_disable_dlclose() false
 
-#elif MAC_OS_X_VERSION_MIN_REQUIRED >= MAC_OS_X_VERSION_10_11
+#elif MAC_OS_X_VERSION_MIN_REQUIRED >= 101100
 /* targeting newer versions only */
 # define dln_disable_dlclose() false
 
 #elif !defined(MAC_OS_X_VERSION_10_11) || \
-    (MAC_OS_X_VERSION_MAX_ALLOWED < MAC_OS_X_VERSION_10_11)
+    (MAC_OS_X_VERSION_MAX_ALLOWED < 101100)
```

Comment:
I see this changed in a couple of places, but we already asserted that it must be defined on the previous line, so should we use the symbolic constant?

Reply:
Did very old `clang -Wundef` warn this usage?

Reply:
@nobu @ioquatix Well, I have dropped the commit changing macOS versions to numeric ones, for now.